### PR TITLE
make issue publishing async

### PR DESCRIPTION
**Pull request by everreau**
- make issue publishing async
- show publishing history on issue plugin page
- add docs

### CODE REVIEW

File: templates/eschol/published.html [hunk 1]
```diff
@@ -15,30 +15,21 @@
...
-        <ul>
-            {% for e in errors %}
-                <li>{{ e }}</li>
-            {% endfor %}
-        </ul>
-    </div>
-    {% endif %}
-
-    {% if objs|length > 0 %}
+    {% if pub_history.success %}
```
Comment (myucekul) [inline templates/eschol/published.html:18]: Am I reading this correctly. If publication history is not successful, then we don't want to show the history.

**Review by myucekul — APPROVED** [review]

File: views.py [hunk 2]
```diff
@@ -1,61 +1,66 @@
...
-    epubs, errors = issue_to_eschol(request=request, issue=issue)
-    context = {
-        'plugin_name': PLUGIN_NAME,
-        'obj': issue,
-        'objs': epubs,
-        'errors': errors,
-        'issue': issue,
-        'obj_name': "Issue"
-    }
+    async_task('plugins.eschol.views.publish_issue_task', issue_id, group=issue_id)
```
Comment (bibliotechy) [inline views.py:31]: having to specify the exact path like that feels weird and brittle. Is that because this is a plugin and therefore, weird?

**Review by bibliotechy — APPROVED** [review]
Love all of the tests and linting! 

Overall it makes sense to me, just one question in line about how the function passed to the `async_task()` is called, but not a blocker.

File: templates/eschol/published.html (hunk 1 above)
Comment (everreau) [reply templates/eschol/published.html:18]: @myucekul it's shown either way but it's different if it's not successful it's in an "alert" markup.  If it's successful it's with success but also, it's still possible that EZID failed so it'll also display that as an "alert" if applicable.

File: views.py (hunk 2 above)
Comment (everreau) [reply views.py]: @bibliotechy you're right there is a better way to do this. I've updated. (Though, there are other places in plugins in general that require a plugin to be in a directory with a particular name but no need to add more)

**Pull request merged**